### DISCUSSION TITLE
Error accessing GONG H-alpha files

### DISCUSSION THREAD

**Discussion by takucera** [Helioviewer-Project/JHelioviewer-SWHV#234] [Q&A]
Hi, I am trying to look at some 2015 GONG data from ROB and am getting the error below.  

Error encountered during JPX creation: Error creating JPX file
COMMAND:
PATH="" /usr/local/bin/kdu_merge_moved -s /dev/stdin
ARGUMENTS:
-i /data/jp2/gong_halpha/2015/02/01/2015_02_01__10_01_54__NSO-GONG_GONG_H-alpha_6562.jp2,/data/jp2/gong_halpha/2015/02/01/2015_02_01__12_02_34__NSO-GONG_GONG_H-alpha_6562.jp2,/data/jp2/gong_halpha/2015/02/01/2015_02_01__14_03_34__NSO-GONG_GONG_H-alpha_6562.jp2,/data/jp2
....[editing out long file list]...
RETURN VALUE: 255
OUTPUT:
Error in Kakadu File Format Support:
Unable to open input file,
"/data/jp2/gong_halpha/2015/02/01/2015_02_01__10_01_54__NSO-GONG_GONG_H-alpha_6
562.jp2".

**Comment by bogdanni** [comment]
GONG H-alpha available again in JHV 4.6 or later since 2024-02-06.
